opentera-webrtc-native-client/OpenteraWebrtcNativeClient/src/Signaling/WebSocketSignalingClient.cpp

```diff
@@ -9,7 +9,7 @@ using namespace std;
 
 constexpr int SignalingProtocolVersion = 1;
```

Comment:
I think this should be changed since it is not compatible with previous protocol v1.

Reply:
Can we fix the Mac test that sometimes fail ?